scripts.js

```diff
@@ -131,144 +124,85 @@ function getRandomIndex(array) {
+    colorBox1.style.backgroundColor = `${savedPalettes[i].firstColor.hexCode}`;
     var colorBox2 = document.createElement("div");
     colorBox2.classList.add("saved-color");
-    savedPalette.appendChild(colorBox2)
-    colorBox2.style.backgroundColor = `${savedPalettes[i].secondColor.hexCode}`
+    savedPalette.appendChild(colorBox2);
+    colorBox2.style.backgroundColor = `${savedPalettes[i].secondColor.hexCode}`;
     var colorBox3 = document.createElement("div");
     colorBox3.classList.add("saved-color");
-    savedPalette.appendChild(colorBox3)
-    colorBox3.style.backgroundColor = `${savedPalettes[i].thirdColor.hexCode}`
+    savedPalette.appendChild(colorBox3);
+    colorBox3.style.backgroundColor = `${savedPalettes[i].thirdColor.hexCode}`;
     var colorBox4 = document.createElement("div");
     colorBox4.classList.add("saved-color");
-    savedPalette.appendChild(colorBox4)
-    colorBox4.style.backgroundColor = `${savedPalettes[i].fourthColor.hexCode}`
+    savedPalette.appendChild(colorBox4);
+    colorBox4.style.backgroundColor = `${savedPalettes[i].fourthColor.hexCode}`;
     var colorBox5 = document.createElement("div");
     colorBox5.classList.add("saved-color");
-    savedPalette.appendChild(colorBox5)
-    colorBox5.style.backgroundColor = `${savedPalettes[i].fifthColor.hexCode}`
+    savedPalette.appendChild(colorBox5);
+    colorBox5.style.backgroundColor = `${savedPalettes[i].fifthColor.hexCode}`;
     var trash = document.createElement("div");
     trash.classList.add("trash-icon");
-    savedPalette.appendChild(trash)
+    trash.setAttribute("id", `${savedPalettes[i].id}`);
+    savedPalette.appendChild(trash);
     }
-
-}
-
-// data-hexcode="${savedPalettes[i].firstColor.hexCode}"
-
-function deletePalette(event) {
-   if (event.target.className === '.trash') {
-
-   }
-}
-
-// function generateInitialColors() {
-//   color1 = new Color(generateHexCode())
-//   colorOneDiv.style.backgroundColor = color1.hexCode
-//   hexCodeOne.innerText = color1.hexCode
-//   color2 = new Color(generateHexCode())
-//   colorTwoDiv.style.backgroundColor = color2.hexCode
-//   hexCodeTwo.innerText = color2.hexCode
-//   color3 = new Color(generateHexCode())
-//   colorThreeDiv.style.backgroundColor = color3.hexCode
-//   hexCodeThree.innerText = color3.hexCode
-//   color4 = new Color(generateHexCode())
-//   colorFourDiv.style.backgroundColor = color4.hexCode
-//   hexCodeFour.innerText = color4.hexCode
-//   color5 = new Color(generateHexCode())
-//   colorFiveDiv.style.backgroundColor = color5.hexCode
-//   hexCodeFive.innerText = color5.hexCode
-// }
+};
 
 function generateColors() {
   if (color1.locked === false) {
-    color1 = new Color(generateHexCode())
-  colorOneDiv.style.backgroundColor = color1.hexCode
-  hexCodeOne.innerText = color1.hexCode
-}
+    color1 = new Color(generateHexCode());
+    colorOneDiv.style.backgroundColor = color1.hexCode;
+    hexCodeOne.innerText = color1.hexCode;
+  }
+
```

Comment:
This is a case where things _could_ be refactored but I don't think it's super worth your time here. The main reason being that you're doing 5 comparisons, 5 style changes, and 5 html changes all in the same function, not 1 comparison, 1 style change, and 1 html change called 5 times. It's hard to refactor that. It's tough to elaborate on the _why_ in PR comments, but this is a case where I think "improving" this is pretty dismal return on investment in terms of time. Part of the process of Turing right now is discovering and gaining an understanding of some of the challenges and pain points of writing code in vanilla javascript, html, and css. A lot of this redundant but somewhat unavoidable code-writing goes away when you start learning better tools for DOM manipulation in Mod 3.